doc/user_quide.asciidoc

```diff
@@ -0,0 +1,105 @@
+# SUSE Distribution Migration System
+===========
+:Authors: Marcus Schäfer and Jesús Bermúdez Velázquez
+:Publication_Date: TBD
+:Latest_Version: 0.5.15
+:Contributors: 
+:Repo: https://github.com/SUSE/suse-migration-services[suse-migration-services]
+
+[%hardbreaks]
+Authors: {Authors}
+Contributors: {Contributors}
+Publication Date: {Publication_Date}
+Latest Version: {Latest_Version}
+Code available: {Repo}
+
+=== Concept
+Upgrading from one release to a newer one having to skip a range of multiple
+releases introduces an upgrade problem.
+
+The Distribution Migration System provides an upgrade path from A to B,
+where A and B are to major distributions versions (for example, from SLES 12 SP4 to SLES 15 GA).
+
+=== Requirements
+The system to be migrated must be registered.
```

Comment:
must be registered to SCC, SMT or RMT.

(SUSE Manager and custom repo configs are not "yet" supported)